plc4j/integrations/apache-nifi/nifi-plc4x-processors/src/main/java/org/apache/plc4x/nifi/record/Plc4xReadResponseRecordSet.java

```diff
@@ -58,9 +58,10 @@ public Plc4xReadResponseRecordSet(final PlcReadResponse readResponse) throws IOE
         rsColumnNames = responseDataStructure.keySet();
                
         if (recordSchema == null) {
```

Comment:
Could we not cache here as well?

Reply:
This part is already cached on the Plc4xSourceRecordProcessor L187 (https://github.com/zylklab/plc4x/blob/4871a0ca4fd3befc773fd624a33ef31372647751/plc4j/integrations/apache-nifi/nifi-plc4x-processors/src/main/java/org/apache/plc4x/nifi/Plc4xSourceRecordProcessor.java#L187)